chart/values.yaml

```diff
@@ -53,7 +53,7 @@ pruner:
   image:
     # Container image for Tekton pruner. Defaults to gcr.
     repository: "gcr.io/tekton-releases/dogfooding/tkn"
-    tag: "a572f1748eb5c0e8002dd0a83ee836ad9d755d035cbd5bfbdef5de810d2bea0b"
+    tag: "sha256:a572f1748eb5c0e8002dd0a83ee836ad9d755d035cbd5bfbdef5de810d2bea0b"
```

Comment:
@cmorinupgrade seems under template, we are generating the complete image with `:`
we need to use `@`, if we go with sha
can you please update and merge the commits. thanks!